modules/howtos/pages/collecting-information-and-logging.adoc

```diff
@@ -4,56 +4,51 @@
 :navtitle: Collecting Information
 :page-aliases: logging
 :page-aliases: event-bus-metrics,logging
-:page-status: UNDER CONSTRUCTION
 
 include::project-docs:partial$attributes.adoc[]
 
 [abstract]
 
 {description}
 
-WARNING: the Logging implementation has changed substantially in SDK 4.0 and is not currently fully documented.
-This will be resolved in a future 4.x release.
+The Python SDK allows logging via the standard `logging` module.
 
-The Python SDK allows logging via the `PYCBC_LOG_LEVEL` environment variable.
-// TODO: Add this back when it is available and remove the line above.
-// The Python SDK allows logging via the standard `logging` module, or via the `PYCBC_LOG_LEVEL` environment variable.
+== Enabling Logging
 
-// == Enabling Logging
-
-// .Python Logging Module
-// [source,python]
-// ----
-// include::example$logging_example.py[tag=logging,indent=0]
-// ----
+.Python Logging Module
+[source,python]
+----
+include::example$logging_example.py[tag=logging,indent=0]
+----
 
-=== Environmental Settings
+// PYCBC_LOG_LEVEL is not utilized as of 4.0.2, maybe at a later date
```

Comment:
Thanks @thejcfactor - would be good to know if/when we bring this back so we can let users know somehow.
For now I'm happy to leave this commented out though.

Reply:
Will do!  I think it might be good to bring back in the event logging hooks are not in play for an app.  Not something we'd recommend, but in that scenario setting an ENV variable would be easier.